sktime/forecasting/online_ensemble.py

```diff
@@ -0,0 +1,171 @@
+            self.ensemble_algorithm = EnsembleAlgorithms(len(forecasters))
+        else:
+            self.ensemble_algorithm = ensemble_algorithm
+
+#         if self.ensemble_algorithm.n != len(forecasters):
+#             raise ValueError("Number of Experts in Ensemble Algorithm \
+#                              doesn't equal number of Forecasters")
+
+        super(EnsembleForecaster, self).__init__(forecasters=forecasters,
+                                                 n_jobs=n_jobs)
+
+    def _fit_ensemble(self, y_val, X_val=None):
+        """Fits the ensemble by allowing forecasters to predict and
+           compares to the actual parameters.
+
+        Parameters
+        ----------
+        y_val : pd.Series
+            Target time series to which to fit the forecaster.
+        X_val : pd.DataFrame, optional (default=None)
+            Exogenous variables are ignored
+        """
+        fh = np.arange(len(y_val)) + 1
+        expert_predictions = np.column_stack(self._predict_forecasters(
+                                             fh=fh, X=X_val))
+        actual_values = np.array(y_val)
+
+        self.ensemble_algorithm._update(expert_predictions.T, actual_values)
+
+    def update(self, y_new, X_new=None, update_params=False):
+        """Update fitted paramters and performs a new ensemble fit.
+
+        Parameters
+        ----------
+        y_new : pd.Series
+        X_new : pd.DataFrame
+        update_params : bool, optional (default=False)
+
+        Returns
+        -------
+        self : an instance of self
+        """
+        self._fit_ensemble(y_new, X_val=X_new)
+
+        self.check_is_fitted()
+        self._set_oh(y_new)
+        for forecaster in self.forecasters_:
+            forecaster.update(y_new, X_new=X_new, update_params=update_params)
+
+        return self
+
+    def update_predict(self, y_test, X_test=None, update_params=False,
+                       return_pred_int=False,
+                       alpha=DEFAULT_ALPHA):
+        """Make and update predictions iteratively over the test set.
+
+        Parameters
+        ----------
+        y_test : pd.Series
+        cv : temporal cross-validation generator, optional (default=None)
+        X_test : pd.DataFrame, optional (default=None)
+        update_params : bool, optional (default=False)
+        return_pred_int : bool, optional (default=False)
+        alpha : int or list of ints, optional (default=None)
+
+        Returns
+        -------
+        y_pred : pd.Series
+            Point predictions
+        y_pred_int : pd.DataFrame
+            Prediction intervals
+        """
+
+        return self._predict_moving_cutoff(y_test, X=X_test,
+                                           update_params=update_params,
+                                           return_pred_int=return_pred_int,
+                                           alpha=alpha)
+
+    def _predict_moving_cutoff(self, y, cv=OnlineSplitter(), X=None,
```

Comment:
Have you changed anything in this method apart from the default input arg for cv?

Reply:
It was that and setting `fh = np.array([1])`. But if I'm using `SlidingWindowSplitter(start_with_window=True, window_length=1, fh=1)` then it would basically just be changing the default input arg. Is there an easy way to set the default?

Reply:
Since this method is used only internally, you could just pass these arguments to it wherever it's used.